.storybook/components/EditEvent/DurationPicker.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react"
+import { Button, View } from "react-native"
+import { EditEventDurationPickerView } from "@edit-event/DurationPicker"
+import { GestureHandlerRootView } from "react-native-gesture-handler"
+import { Provider, atom } from "jotai"
+
+const EditEventDurationsMeta = {
+  title: "Edit Event Durations"
+}
+
+export default EditEventDurationsMeta
+
+const PRESETS = [
+  300 * 15,
+  601 * 15,
+  900 * 15,
+  1201 * 15,
+  3601 * 15,
+  5400 * 15
```

Comment:
Why are these multiplied by 15? If they're presets, wouldn't they be multiplied by a factor of 60 to represent minutes or hour intervals?

Reply:
This was another test in storybook, this time to ensure that the text size of the formatted durations fit nicely in the slider.